tasks/ci.rake

```diff
@@ -182,6 +182,17 @@ EOS
       beaker_suite(:aio)
     end
 
+    desc <<-EOS
+Run the acceptance tests using puppet-agent (AIO) packages with retries.
+
+  $ SHA=<full sha> bundle exec rake ci:test:aioretry
+
+SHA should be the full SHA for the puppet-agent package.
+EOS
+    task :aioretry => ['ci:check_env', 'ci:gen_hosts'] do
+      beaker_suite_retry(:aio)
```

Comment:
This is a task that wraps a single method -- would it make sense to just have the retry toggle be a second argument to `beaker_suite`?

Reply:
agreed, seems like doing this would avoid some duplicate code in the methods as well

Reply:
I think the control flow is sufficiently different that trying to conditionalize (is that a word?) it within one method makes things much less clear than having discrete methods. I agree with don't need a task each; adding a defaulted parameter to :aio would address that.

Reply:
Defaulted parameter to :aio added.